src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -22,6 +22,8 @@
 #include "assets/objects/gameplay_keep/gameplay_keep.h"
 #include "assets/objects/object_link_child/object_link_child.h"
 
+#include "config.h"
```

Comment:
is this a left-over? GitHub doesn't show me more changes in this file

Reply:
Yep, when I removed the faster bunnyhood, I forgot this include